lib/licensee/projects/fs_project.rb

```diff
@@ -34,7 +34,10 @@ def initialize(path, **args)
       def files
         @files ||= search_directories.flat_map do |dir|
           relative_dir = Pathname.new(dir).relative_path_from(dir_path).to_s
-          Dir.glob(::File.join(dir, @pattern).tr('\\', '/')).map do |file|
+
+          # Hidden files are excluded by default. Explicitly include IgnoreFile in file list
```

Comment:
I don't think I understand what "Hidden files are excluded by default" means. Not recalling concept of a hidden file being used in licensee.

Reply:
I found a better implementation that didn't require the comment. In short, Ruby's `Dir.glob` command ignores Unix-style hidden files (e.g., `.licensee.yml`), which you need to instruct it to include, via the `File::FNM_DOTMATCH` flag).